controller/workitem_blackbox_test.go

```diff
@@ -1288,6 +1288,35 @@ func (s *WorkItem2Suite) TestWI2SuccessCreateWorkItemWithDescriptionAndMarkup()
 	assert.Equal(s.T(), rendering.SystemMarkupMarkdown, wi.Data.Attributes[workitem.SystemDescriptionMarkup])
 }
 
+func (s *WorkItem2Suite) TestWI2SuccessUpdateWorkItemWithDescriptionAndMarkup() {
+	// given
+	c := minimumRequiredCreatePayload()
+	c.Data.Attributes[workitem.SystemTitle] = "Title"
+	c.Data.Attributes[workitem.SystemDescription] = rendering.NewMarkupContent("Description", rendering.SystemMarkupMarkdown)
+	c.Data.Relationships.BaseType = newRelationBaseType(workitem.SystemBug)
+
+	// when
+	_, wi := test.CreateWorkitemsCreated(s.T(), s.svc.Context, s.svc, s.workitemsCtrl, *c.Data.Relationships.Space.Data.ID, &c)
+	assert.Equal(s.T(), "Description", wi.Data.Attributes[workitem.SystemDescription])
+	assert.Equal(s.T(), "<p>Description</p>\n", wi.Data.Attributes[workitem.SystemDescriptionRendered])
```

Comment:
there is no HTML escape in the expected output, is there ? could you change the input (markdown) so that the output shows some sort of HTML escaping ?

Reply:
No there is no HTML escaping because this is plainText. I've updated this in https://github.com/fabric8-services/fabric8-wit/pull/2165/commits/f3f8b7e6aae2dbc3c89930923a72ff02328b5f0d